ride_share.rb

```diff
@@ -0,0 +1,60 @@
+rides = {
+    DR0004: [
+      {date: "3rd Feb 2016", cost: 5, rider_id: "RD0022", rating: 5},
+      {date: "4th Feb 2016", cost: 10, rider_id: "RD0022", rating: 4}, 
+      {date: "5th Feb 2016", cost: 20, rider_id: "RD0073", rating: 5}
+    ],
+    DR0003: [
+      {date: "4th Feb 2016", cost: 5, rider_id: "RD0066,", rating: 5},
+      {date: "5th Feb 2016", cost: 50, rider_id: "RD0003", rating: 2}
+    ],
+    DR0002: [
+      {date: "3rd Feb 2016", cost: 25, rider_id: "RD0073", rating: 5}, 
+      {date: "4th Feb 2016", cost: 15, rider_id: "RD0013", rating: 1},
+      {date: "5th Feb 2016", cost: 35, rider_id: "RD0066", rating: 3}
+    ],
+    DR0001: [
```

Comment:
You are missing a ride for DR0001. As a check, make sure the number of rides you have matches the number of rows in the csv.